src/Suggestor/Suggestor.ts

```diff
@@ -98,6 +98,14 @@ function getPossibleComponentSuggestions(
             displayText: `${symbols.scheduledDateSymbol} scheduled date`,
             appendText: `${symbols.scheduledDateSymbol} `,
         });
+    if (!line.includes(symbols.createdDateSymbol)) {
+        const parsedDate = DateParser.parseDate('today', true);
+        const formattedDate = parsedDate.format(TaskRegularExpressions.dateFormat);
+        suggestions.push({
+            displayText: `${symbols.createdDateSymbol} created today (${formattedDate})`,
+            appendText: `${symbols.createdDateSymbol} ${formattedDate} `,
+        });
+    }
```

Comment:
I tentatively put it after date-like suggestions and before priority, but I'm happy to move it!
Also, all the date suggestions include the computed date in parentheses so I replicated the pattern here too, but I'm not actually sure it's very useful.